build/common.ps1

```diff
@@ -204,6 +205,7 @@ Function Install-DotnetCLI {
     }
 
     # Install the 2.x runtime because our tests target netcoreapp2x
+    Trace-Log "$DotNetInstall -Runtime dotnet -Channel 2.2 -i $CLIRoot -NoPath"
     & $DotNetInstall -Runtime dotnet -Channel 2.2 -i $CLIRoot -NoPath
```

Comment:
Are `$CLIRoot` and `$cli.Root` the same locations ?

Reply:
yes: https://github.com/NuGet/NuGet.Client/blob/2bddc10887d1ebfb5473ecc9fc234578cc995109/build/common.ps1#L175